types/royalslider/index.d.ts

```diff
@@ -367,6 +367,73 @@ declare namespace RoyalSlider {
         visibleNearby?: RoyalSliderVisibleOptions;
     }
 
+    export interface RoyalSliderSlide {
+        /**
+         * TODO
```

Comment:
This will show up in Quick Info in the editor so it might be preferable to omit it until actual documentation is available.